src/Npgsql/PostgresErrorCodes.cs

```diff
@@ -0,0 +1,468 @@
+﻿namespace Npgsql
+{
+    /// <summary>
+    /// Provides constants for PostgreSQL error codes.
+    /// </summary>
+    /// <remarks>
+    /// See http://www.postgresql.org/docs/current/static/errcodes-appendix.html
+    /// </remarks>
+    public static class PostgresErrorCodes
+    {
+        #pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Move it outside of the namespace block to keep the common style. For example, see [`NpgsqlRawCopyStream.cs`](https://github.com/npgsql/npgsql/blob/25469adb61ad66aed227008476d65f19833433d6/src/Npgsql/NpgsqlRawCopyStream.cs#L35).